apps/src/lib/kits/maker/ui/EligibilityChecklist.jsx

```diff
@@ -170,9 +179,12 @@ export default class EligibilityChecklist extends React.Component {
         {this.state.statusYear === Status.SUCCEEDED && (
           <div>
             <div>
+              You meet all the requirements for a fully subsidized classroom
+              kit. Click the “Get Code” button to get your code.
+              <br />
               <strong>
-                You meet all the requirements for a fully subsidized classroom
-                kit. Click the “Get Code” button to get your code.
+                If you are in Alaska or Hawaii, please contact teacher@code.org
+                to receive a discount code that covers shipping.
```

Comment:
This change won't be necessary once we have the automated state detection added.

Reply:
Fixed [here](https://github.com/code-dot-org/code-dot-org/pull/32179/commits/e573d65df4e443603d8d752f05d1e609eaecf983)